modules/administration_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -453,22 +453,49 @@ sudo -u www-data php occ config:system:set \
 Value not updated, as it has not been set before.
 ....
 
-Note that in order to write a Boolean, float, or integer value to the
-configuration file, you need to specify the type on your command. This
-applies only to the `config:system:set` command.
+Note that in order to write a boolean, float, JSON, or integer value to the configuration file, you need to specify the type on your command.
+This applies only to the `config:system:set` command.
+The following values are known:
 
-When you want to e.g., disable the maintenance mode run the following
-command:
+* `boolean`
+* `integer`
+* `float`
+* `json`
+* `string` (default)
+
+Examples
+
+Disable the maintenance mode:
 
 ....
-sudo -u www-data php occ config:system:set \
-   maintenance \
+sudo -u www-data php occ config:system:set maintenance \
    --value=false \
    --type=boolean
+
 ownCloud is in maintenance mode - no app have been loaded
 System config value maintenance set to boolean false
 ....
 
+Create the `app_paths` config setting (using a JSON payload):
+
+[source,console]
+....
+sudo -u www-data php occ config:system:set apps_paths \
+      --type=json \
+      --value='[
+        {
+            "path":"/var/www/owncloud/apps",
+            "url":"apps",
+            "writable":1
```

Comment:
even it works with ``0`` or ``1``
we have in all othe documents ``true`` or ``false``...
in the particular path for ``apps`` it is ``false``
for ``apps-external`` it is ``true``
which maches the othe docs too...

Reply:
Right you are. Thanks for spotting that one.

Reply:
This has been ammended in 21aa9b7.